examples/slides/src/components/Thanks.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import Link from './Link';
+import './Thanks.css';
+import meme from '../assets/tomhanksmeme.jpg';
+
+export default class Thanks extends React.Component {
+  static MAX_HANKS = 40;
+
+  constructor() {
+    super();
+    this.state.message = 'T.hanks';
+    this.state.numOfHanks = 1;
+  }
+
+  clickHandler = () => {
+    if (this.state.numOfHanks < Thanks.MAX_HANKS) {
+      this.setState({
+        message: 'T.hanks a lot!',
+        numOfHanks: this.state.numOfHanks * 2,
+      });
+    }
+  };
+
+  render() {
+    const HanksByRow = [];
+    for (let i = 1; i <= this.state.numOfHanks; i++) {
+      const currHanksRow = [];
+      for (let j = 1; j <= this.state.numOfHanks; j *= 2) {
+        currHanksRow.push(
```

Comment:
nit: prefer lowerCamelCase for var names
nit: prefer doing maps in the render block rather than accumulating data before